static/js/tasks_browse.js

```diff
@@ -288,6 +288,103 @@ $(document).ready(function() {
+        bookmarksBody.empty()
+        bookmarks.forEach(bookmark => {
+            let row = $("<tr>").appendTo(bookmarksBody);
+            row.append(
+                $("<td>").append(
+                    $("<a>", {
+                        "href": bookmark["url"],
+                        "class": "label label-info",
+                        "text": bookmark["name"]
+                    })
+                )
+            )
+            row.append(
+                $("<td>").append(
+                    $("<a>", {
+                        "class": "label label-danger delete-bookmark",
+                        "text": "Delete",
+                        "data-name": bookmark["name"]
+                    })
+                )
+            )
+        })
+        $('.delete-bookmark').on('click', function(e){
+            deleteBookmark($(this).data('name').toString())
+        })
+    }
+
+    function getNumBookmarks() {
+        bookmarksBody = $('#bookmarksGrid > tbody')
+        return bookmarksBody.children().length
+    }
+
+    function getTaskbrowseBookmarksUrl() {
+        let url_arr = window.location.pathname.split('/')
+        let username = $('#currentUsername').text()
+        let project = url_arr[url_arr.indexOf('project') + 1]
+
+        let url = window.location.origin + '/account/' + username + '/taskbrowse_bookmarks/' + project + '?order_by=' + bookmark_sort + '&desc=' + bookmark_desc
+        return url
+    }
+
+    function getBookmarks() {
+        sendGetRequest(getTaskbrowseBookmarksUrl(), null).done(function(res) {
+            showBookmarks(res)
+        });
+    }
+
+    $('#add-bookmark').click(() => {
+        const MAX_TASKBROWSE_BOOKMARKS = 100
+        var modal = $('#addBookmarkModal');
+        if (getNumBookmarks() >= MAX_TASKBROWSE_BOOKMARKS) {
+            alert("Exceeded limit of 100 bookmarks per project!")
+        }
+        else {
+            nameInput = $('.modal-body #bookmark-name')
+            data = {
+                "name": nameInput.val(),
+                "url": window.location.href
+            }
+            sendUpdateRequest(getTaskbrowseBookmarksUrl(), data).done(function(res) {
+                showBookmarks(res)
+                nameInput.val('')
+                setSpinner(false)
+            });
+        }
+        modal.modal('hide');
+    });
+
+
+    function deleteBookmark(name) {
+        data = {
+            "name": name,
+        }
+        sendDeleteRequest(getTaskbrowseBookmarksUrl(), data).done(function(res) {
+            showBookmarks(res)
+            setSpinner(false)
+        });
+    }
+
+    $('.delete-bookmark').on('click', function(e){
+        deleteBookmark($(this).data('name'))
+    })
+
+    let bookmark_sort = "name"
+    let bookmark_desc = false
+
+    $('#sort-bookmarks').on( "change", function() {
+        bookmark_sort = $(this).val()
+
+        if (bookmark_sort == "name") bookmark_desc = false
+        else bookmark_desc = true
```

Comment:
Could you re-rewrite these two lines with a better format? Also consider using "===" with string comparison in JS. One-liner is also possible, but might sacrifice some readability.